### ISSUE TITLE
nDAX/nCAT can't find station names with spaces

### ISSUE THREAD

**Issue by tmiw** [arodland/nDAX#14]
According to SmartSDR for Mac, the station name for my machine is supposedly "Mooneer's 16" MacBook Pro" as shown in the below screenshot:

<img width="617" alt="image" src="https://github.com/user-attachments/assets/6cbbf08f-fa68-4677-9214-4d4d7263bc43">

However, when I try to use that in both nDAX and nCAT (inside an Ubuntu 24.04 VM), neither is able to find my running SmartSDR session:

```
mooneer@ubuntu-vm:~/nDAX$ ./nDAX-linux-amd64 -slice A -daxch 1 -source flex.sliceA.rx5 -sink flex.sliceA.tx5 -station Mooneer\'s\ 16\"\ MacBook\ Pro -radio 10.0.1.111
6:26PM INF Waiting for station station="Mooneer's 16\" MacBook Pro"
Terminated
mooneer@ubuntu-vm:~/nDAX$
```

To make sure that's actually the correct station ID, I went looking in the code where that log entry's printed and added some additional debugging (last line below):

```go
func bindClient() {
...
        for !found || !cmdComplete {
                select {
                case upd := <-clients:
                        log.Info().Str("station", upd.CurrentState["station"]).Msg("station info")
```

which resulted in the following:

```
mooneer@ubuntu-vm:~/nCAT$ ./nCAT -station "Mooneer's 16\" MacBook Pro"
6:41PM INF Waiting for station station="Mooneer's 16\" MacBook Pro"
6:41PM INF station info station="Mooneer's\x7f16\"\x7fMacBook\x7fPro"
```

Basically, SmartSDR for Mac seems to be using character code 7F (??) instead of space (either that, or something in nDAX and nCAT is converting the space character to 7F). I'm sure this isn't that uncommon of an occurrence due to Apple defaulting to "[name]'s [product]" for the computer name, so I figured I'd let you know. :)

**Comment by arodland** [comment]
Gotcha. That's a Flex thing, they replace space with 0x7F in various fields that go into their key-value list format, because the list is space-delimited. I've had to deal with it in one or two places in the API, but it never occurred to me that it might apply here.

Thanks for the report, I'll get a fix in soon.
